scripts/upgrade_admin_server.sh

```diff
@@ -77,6 +77,15 @@ upgrade_admin_server()
     # Signalize that the upgrade correctly ended
     echo "12.2" >> $UPGRADEDIR/admin-server-upgraded-ok
 
+    # delete current OS values
+    knife exec -E "n = nodes.find(:roles => 'provisioner-server').first
+n.target_platform = nil
+n.provisioner.default_os = nil
+n.save"
+
+    # let the provisioner fill the new OS values
+    chef-client
+
```

Comment:
Are you suggesting to use `suse-12.2` values directly or just asking what is better?

Chef-client run does set the values correctly and does not seem to do any harm.